keyboards/arteusx/readme.md

```diff
@@ -0,0 +1,4 @@
+ArteusX
```

Comment:
This file should be based on the current template [here](https://github.com/qmk/qmk_firmware/blob/07c6c4acb93b2faefb504bfdc076bc35134f0bee/quantum/template/avr/readme.md).